README.md

```diff
@@ -1,2 +1,28 @@
-# opentelemetry
-Get started with our Elastic Distros of OpenTelemetry
+# Elastic OpenTelemetry Distributions
```

Comment:
This title doesn't have to stay. I wasn't sure what we would want to call this @mlunadia

Reply:
Elastic Distributions of OpenTelemetry

Reply:
@mlunadia can we decide [in this (internal) doc](https://docs.google.com/document/d/1W0pvuEdVQpa8_dpiBBO0mQwZXZwTgBu8S4SkV40kjJQ/edit?disco=AAABL3qFoNY) whether we use "for" or "of". I'd like to establish a single source of truth there, and then @mdbirnstiehl and I can make sure we're using terminology correctly across all docs.

Reply:
>opentelemetry

OpenTelemetry

Reply:
>Elastic Distros of OpenTelemetry

Elastic Distributions of OpenTelemetry